<launch>
   <node pkg="laser_tri" name="laser_split_and_merge_trans" type="laser_split_and_merge_trans" output="screen" respawn="true">
      <param name="goal_range" type="double" value="3" />
      <param name="tri_len" type="double" value="0.53" />
      <param name="tri_len_err" type="double" value="0.07" />
      <param name="angleDif" type="double" value="2" />
      <param name="angleDif_err" type="double" value="0.3" />
      <param name="disDif" type="double" value="0.04" />
      <param name="minRegionPointNum" type="int" value="10" />
      <param name="threshold" type="double" value="0.02" />
   </node>
</launch>
